packages/lib/vue-components/.storybook/config.js

```diff
@@ -4,7 +4,7 @@ import { addReadme } from 'storybook-readme/vue';
 addDecorator(addReadme);
 
 // automatically import all files ending in *.stories.js
-const req = require.context('../src', true, /story\.tsx?$/);
+const req = require.context('../', true, /\.stories\.js$/);
```

Comment:
Please take a look here https://github.com/pyxismedia/laboratory/blob/87880fc624a0c33a0dc74eb6369a12b43fdefc8b/packages/lib/components/.storybook/config.js#L61 it was correct and it does not need change. Components will be per directory and each directory will contain component with story beside each other.

Reply:
btw i suggest to not push into your master because now you will have to revert. Better to create branches ;)